test/transferSpec.js

```diff
@@ -161,6 +143,18 @@ describe('Plugin transfers (optimistic)', function () {
       }, transferA)).catch(done)
     })
 
+    it('should reject optimistic transfer with amount 0', function (done) {
```

Comment:
See previous comment. Support for 0-amount transfers should definitely be allowed, at least.